contribs/gmf/src/directives/filterselector.js

```diff
@@ -2,7 +2,7 @@ goog.provide('gmf.filterselectorComponent');
 
 goog.require('gmf');
 /** @suppress {extraRequire} */
-goog.require('gmf.Authentication');
+goog.require('gmf.authentication.module');
```

Comment:
Oh, I see, it's for the gmfUser. Definitely, it should be moved in another file. perhaps it's own file like the https://github.com/camptocamp/ngeo/blob/master/src/map/FeatureOverlay.js Object.

And then we require only the user, not the whole module.
(Also in another files with the same situation)